logger_node:
  ros__parameters:
    record_wheel_current: True
    record_wheel_voltage: True
